sdk/monitor/Azure.ResourceManager.Monitor/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0035:Output model type should have a corresponding method in a model factory class", Justification = "Generated ARM resource types", Scope = "type", Target = "~T:Azure.ResourceManager.Monitor.DiagnosticSettingResource")]
```

Comment:
check which of these exclusions are still needed given the new analyzer package.